app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
 
+    people = {}
 
-def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+
+        Person.people[name] = self
+
+
+def create_person_list(people: list[dict]) -> list:
+    for person in people:
+        Person(person["name"], person["age"])
+    for person in people:
+        current_person = Person.people[person["name"]]
+        if person.get("wife"):
+            current_person.wife = Person.people.get(person["wife"])
```

Comment:
There is a potential issue here if `person['wife']` is not found in `Person.people`. Consider adding a check to handle cases where the wife is not found to avoid setting `None` as the wife.